python_modules/dagster/dagster/_core/execution/context/input.py

```diff
@@ -149,6 +152,14 @@ def __del__(self):
         if self._resources_cm and self._resources_contain_cm and not self._cm_scope_entered:
             self._resources_cm.__exit__(None, None, None)  # pylint: disable=no-member
 
+    @property
```

Comment:
This will have to be done more thoughtfully in a separate PR. Not having the instance available to InputContext is an oversight IMO.

Reply:
Yeah - filed this recently: https://github.com/dagster-io/dagster/issues/11016